server_commands/__init__.py

```
@@ -1,14 +1,45 @@
+    for plugin in plugins:
+        bot.add_plugin(plugin)
+
+
+__all__ = [
+    "FunPlugin",
+    "ModerationPlugin",
+    "InfoPlugin",
+    "DEFAULT_PLUGINS",
+    "build_default_plugins",
+    "load_default_plugins",
+]
```

Ruff (0.15.10)

[warning] `__all__` is not sorted

Apply an isort-style sorting to `__all__`

(RUF022)